docs/pywbemcli/commands.rst

```diff
@@ -1534,6 +1549,252 @@ See :ref:`pywbemcli server remove-mof --help` for the exact help output of the
+only exist in some servers and may be optionally enabled and disabled
+differently in servers because while the capabilities were documented in the
+DMTF schemas, they were never included as part of either DMTF or SNIA profile
+requirements so the implementations vary.
+
+Pywbemxli provides a command to activate the statistics gathering based on the
+CIM_CIMObjectManager class GatherStatisticalData property.
+
+The pywbemcli statistics report is table based showing all of the currently
+gathered statistics: The following is an example of this report.
+
+.. code-block:: text
+
+    pywbemcli> statistics show
+    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
+       Op    Exc  Op Time(S)         RequestLen     ReplyLen       Operation
+      Cnt    Cnt  Avg/Min/Max        Avg/Min/Max    Avg/Min/Max
+    -----  -----  -----------------  -------------  -------------  ----------------------
+        2      0  0.045/0.041/0.049  332            63309          EnumerateClassNames
+        3      1  0.019/0.005/0.033  551/536/560    1501/298/3838  OpenEnumerateInstances
+        2      0  0.006/0.006/0.006  542            4187           GetClass
+
+If the WBEM server is enabled to gather statistics and each CIM/XML response
+includes server response time, the report includes the extra row ``Server
+Time``.
+
+.. code-block:: text
+
+    pywbemcli>statistics show
+    Statistics: Time(Seconds); Times/lengths: (Avg/Min/Max), single value if all same.
+       Op    Exc  Op Time(S)         Server Time(S)     RequestLen     ReplyLen        Operation
+      Cnt    Cnt  Avg/Min/Max        Avg/Min/Max        Avg/Min/Max    Avg/Min/Max
+    -----  -----  -----------------  -----------------  -------------  --------------  ----------------------
+        2      0  0.045/0.041/0.049  0.000/inf/0.000    332            63309           EnumerateClassNames
+        2      1  0.026/0.005/0.047  0.000/inf/0.000    333/321/345    6225/298/12152  EnumerateInstanceNames
+        5      2  0.014/0.004/0.033  0.000/0.001/0.001  550/536/560    1728/298/3838   OpenEnumerateInstances
+        1      0  0.013              0.001              3483           224             ModifyInstance
+        2      0  0.010/0.010/0.010  0.000/0.001/0.001  345            3504/3504/3505  EnumerateInstances
+        1      0  0.008              0.001              822            2976            GetInstance
+        2      0  0.006/0.006/0.006  0.000/inf/0.000    542            4187            GetClass
+
+Pywbemcli gathered statistics are reported to the console based on the value of
+the --timestats general option:
+
+a. 'auto' - statistics display each time a command is executed.
+
+b. 'cmd' - display statistics only when the command ``statistics show`` is executed.
+
+The pywbemcli gathered statistics can be cleared with the command ``statistics
+reset``.
+
+Statistics gathered and maintained in the server are only displayed if the
```

Comment:
"gathered and maintained" - this belongs together; it does not make sense to gather and not maintain. Since we now moved to using just "gathered" in most other places, I suggest removing "and maintained".

Reply:
I did maintained since the server has the capability to reset, etc. also.

DONE